srml/democracy/src/lib.rs

```diff
@@ -239,7 +398,8 @@ decl_storage! {
 		pub MinimumDeposit get(minimum_deposit) config(): BalanceOf<T>;
 		/// The delay before enactment for all public referenda.
 		pub PublicDelay get(public_delay) config(): T::BlockNumber;
-		/// The maximum number of additional lock periods a voter may offer to strengthen their vote. Multiples of `PublicDelay`.
+		/// The maximum number of additional lock periods a voter may offer to strengthen
+		/// their vote. Multiples of `PublicDelay`.
```

Comment:
```suggestion
		/// their vote. `PublicDelay` is a multiple of `LockPeriods`.
```

Reply:
i don't think ""Multiples of `Public`" delay makes sense, since:
* `PublicDelay` is measured using `BlockNumber`, which is type `u64`.
* `LockPeriods` is type `i8`.